source/_posts/2024-01-03-release-20241.markdown

```diff
@@ -188,6 +188,33 @@ noteworthy changes this release:
 - Add alarm platform to Comelit ([@chemelli74] - [#104718])
 - Add Support for SleepIQ Foot Warmers ([@kbickar] - [#105931])
 
+[@MartinHjelmare]: https://github.com/MartinHjelmare
+[@mill1000]: https://github.com/mill1000
+[@bdraco]: https://github.com/bdraco
+[@slhad]: https://github.com/slhad
+[@killer0071234]: https://github.com/killer0071234
+[@azogue]: https://github.com/azogue
+[@Anonym-tsk]: https://github.com/Anonym-tsk
+[@iamrgroot]: https://github.com/iamrgroot
+[@jjlawren]: https://github.com/jjlawren
+[@chemelli74]: https://github.com/chemelli74
+[@pfrazer]: https://github.com/pfrazer
+[@kbickar]: https://github.com/kbickar
+[#99608]: https://github.com/home-assistant/core/pull/99608
+[#104874]: https://github.com/home-assistant/core/pull/104874
+[#104881]: https://github.com/home-assistant/core/pull/104881
+[#103781]: https://github.com/home-assistant/core/pull/103781
+[#79077]: https://github.com/home-assistant/core/pull/79077
+[#105454]: https://github.com/home-assistant/core/pull/105454
+[#105494]: https://github.com/home-assistant/core/pull/105494
+[#101102]: https://github.com/home-assistant/core/pull/101102
+[#85769]: https://github.com/home-assistant/core/pull/85769
+[#105741]: https://github.com/home-assistant/core/pull/105741
+[#101374]: https://github.com/home-assistant/core/pull/101374
+[#106290]: https://github.com/home-assistant/core/pull/106290
+[#104718]: https://github.com/home-assistant/core/pull/104718
+[#105931]: https://github.com/home-assistant/core/pull/105931
```

Comment:
These don't belong here.

```suggestion
```

Reply:
Why, they are mentioned above right? Or should they join the bigger list down?

Reply:
No, they are there for processing and aren't processed yet. They will not be linked to PRs.